README.md

```diff
@@ -33,25 +33,25 @@ When creating a new project, you can either copy an existing projects values fil
 Or use `./create-project`. To auto-generate the files for a new project. Run `./create-project project.name` and the tool will create the config files for you.
 
 For example:
-`./create-project ubuntu.com`
+`./create-project new-site.com`
 
 Will create:
- - /sites/ubuntu-com.yaml
+ - /sites/new-site.com.yaml
 
 ### To deploy a new service
 
 E.g. to deploy the snapcraft.io service to staging from scratch:
 
 ``` bash
 # E.g. To deploy the snapcraft.io services to staging
-./konf.py staging sites/snapcraft-io.yaml | kubectl apply --filename -
+./konf.py staging sites/snapcraft.io.yaml | kubectl apply -f -
```

Comment:
I actually prefer `--filename` - expanded options are much more descriptive

Reply:
Okay, I've put it expanded again. I will merge.